PRFCS/prfc-1.md

```diff
@@ -11,7 +11,13 @@ ParallelChain RFC 1 defines a standard interface for fungible tokens implemented
 
 A standard contract interface for fungible tokens allows more seamless interoperability since applications can make the simplifying assumption that all PRFC 1-implementing contracts always export the same, named set of Methods (they may export more).
 
-The below section lists the set of methods that all smart contracts that want to be PRFC 1-compliant must implement, as well as the behaviour that each method must exhibit. Required behaviour involves emitting certain events. These are also listed and described.
+The below sections list the set of methods that all smart contracts that want to be PRFC 1-compliant must implement, as well as the behaviour that each method must exhibit. Required behaviour involves emitting certain events. These are also listed and described.
```

Comment:
Decide if we using UK English "Behaviour" or US English "Behavior", it's "behaviour" in here, but "behavior" in PRFC2

Reply:
Will use "Behavior".